homeassistant/components/whois/__init__.py

```diff
@@ -1 +1,16 @@
-"""The whois component."""
+"""The Whois integration."""
```

Comment:
Two questions (not related to this PR):
 - Should [RDAP](https://www.icann.org/rdap) support be added to this integration, or is that something for a new separate integration?
 - And where should I ask this kind of questions? :smile:

Reply:
I really don't see how this review comment is related to this change set.